### PR TITLE
Kriselda/single product client side

### PR DESCRIPTION
**Pull request by kbonifacio**
One unknown bug with the single product component. Not sure why the if constraint isn't working properly to only map out one product. The id is being pulled correctly from the url and it's being logged. If line 11 is removed, all the data for the products will show (unformatted);

### CODE REVIEW

**Review by richardedwardbrown3 — COMMENTED** [review]
it might be nitpicky, but the variable names 6: allProducts and 15  products seem more accurate to be named: 6 products, 15 product. But not sure if the names were already decided for a specific reason. But all the rest looks great.

**Review by richardedwardbrown3 — APPROVED** [review]

File: routes/products.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const productsRouter = require("express").Router();
 const { getProductById, getAllProducts, createProduct, getProductByName } = require("../db")
-
+//get all products   
 productsRouter.get("/", async (req, res, next) =>{
     try {
         const allProducts = await getAllProducts();
```
Comment (richardedwardbrown3) [inline routes/products.js]: 6: allProducts again to products* ? and notice that product by id is labeled just product.

**Review by richardedwardbrown3 — APPROVED** [review]
love it

**Review by Felix-Cadiz — APPROVED** [review]
LGTM!

**Review by Felix-Cadiz — APPROVED** [review]
LGTM!

**Review by Felix-Cadiz — APPROVED** [review]
Thanks for the updated CSS!

**Pull request merged**